docs/usage/security-modes.md

```diff
@@ -0,0 +1,52 @@
+# Security modes
+
+This section describes the different **Security mode** options for connectivity that can be selected for Liqo, regulating connectivity restriction for intra-cluster traffic.
+
+## Overview
+
+Actually, Liqo provides two security modes:
+
+* **Full pod-to-pod connectivity**: represents the native behaviour of Liqo, there are no restriction in terms of connectivity, and all pods of the first cluster can connect to all pods of the second cluster (and vice versa).
+* **Intra-cluster traffic segregation**: full pod-to-pod connectivity is enabled only within real clusters; regarding intra-cluster traffic, local cluster has the possibility to contact only its offloaded pods, while remote cluster is allowed to contact exclusively services offloaded on it
+
+## Selection of the Security mode
+
+The desired Security mode can be selected with a **feature flag** at install time (see the [reference](/installation/install.md)):
+
+* `--set networking.securityMode=FullPodToPod` to select **full pod-to-pod connectivity (default mode)**
+* `--set networking.securityMode=IntraClusterTrafficSegregation` to select **intra-cluster traffic segregation**
+
+```{admonition} Note
+Currently, the selection of security mode is possible only at installation time. In future, this feature will be extended to be configured at run-time, e.g., each time you setup a new peering.
+```
+
+## Connectivity matrix
+
+To better explain security modes, schemas of possible Liqo topologies and relative traffic matrices are provided in the following.
+<details>
+<summary>Full pod-to-pod connectivity</summary>
+<IMG src="/_static/images/usage/security-modes/full_p2p.svg"/>
+</details>
+
+<details>
+<summary>Intra-cluster traffic segregation</summary>
+
+```{admonition} Note
+Yellow cells in the table highlights the differences compared to the default mode (i.e., full pod-to-pod connectivity).
+```
+
+<blockquote>
```

Comment:
Here we're using a different terminology than before. Here we say "Offloaded pods", "Offloaded service", "Complete scenario". Before, we used "Full pod-to-pod" and "Intra-cluster traffic segregation". Please use the same terminology or clarify what you had in mind.

Reply:
These three pictures are all related to intra-cluster traffic segrgation mode; I would include them because, on one hand, I believe it could be useful to see two separate examples that demonstrate the two types of traffic that the feature allows, making it clear through the summary example and/or in another way that with this security mode, you cannot select only one of them. On the other hand, I think that a page with 3 consecutive images and 3 consecutive tables is not user-friendly at all, and perhaps the explanation of the security mode with just the summary example manages to be explanatory. What do you think?